crates/miden-objects/src/note/recipient.rs

```diff
@@ -59,10 +59,11 @@ impl NoteRecipient {
 
     /// Returns the recipient encoded as [Felt]s.
     pub fn to_elements(&self) -> Vec<Felt> {
```

Comment:
Since it is not really an elements representation of the recipient anymore, I would name it something like `format_for_advice()` by analogy with `NoteInputs`. I would also update the doc comment to show the layout of the returned values, something like 
```suggestion
    /// Returns the recipient formatted to be used with the advice map.
    ///
    /// The format is `INPUTS_COMMITMENT || SCRIPT_ROOT || SERIAL_NUMBER || inputs_length`
    ///
    /// Where: 
    /// - INPUTS_COMMITMENT is the commitment of the note inputs
    /// - SCRIPT_ROOT is the commitment of the note script (i.e., the script's MAST root)
    /// - SERIAL_NUMBER is the recipient's serial number
    /// - inputs_length is the length of the note inputs
    pub fn format_for_advice(&self) -> Vec<Felt> {
```